singer-parent/component-kitap/pom.xml

```diff
@@ -13,181 +13,198 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
```

Comment:
A bit surprised by this change of formatting.
Could we run the spotless formatting on top?

Reply:
Maybe commit formatting change in a separate commit, to make this PR lighter and keep only related to the ticket lines.